http-full-proxy/src/main.rs

```diff
@@ -11,7 +11,7 @@ use actix_web::{
     HttpResponse,
 };
 use clap::{value_t, Arg};
-use futures::Future;
+use futures::{future::ok, Future};
```

Comment:
Do not import `ok` like that, I'd advise to use `future::ok` directly.

Reply:
Got it. Fixed